NEWS.md

```diff
@@ -6,6 +6,19 @@
 
 ## NEW FEATURES
 
+1. New function `fcase(.,default)` has been implemented in C, [#3823](https://github.com/Rdatatable/data.table/issues/3823). It is comparable to `dplyr::case_when`. Please see `?data.table::fcase` for more details.
+
+```R
+# default 4 threads on a laptop with 16GB RAM and 8 logical CPU
+x = sample(1L:10L, 3e8, replace=TRUE) # 1GB
+microbenchmark::microbenchmark(
+	dplyr::case_when(x < 5L ~ 1L, x > 5L ~ 3L),
+	data.table::fcase((x < 5L, 1L, x > 5L, 3L)),
+	times = 5L,
+	unit = "s"
+)
```

Comment:
should include the timing output as a sample right?

Reply:
Yes you are wright. I was wondering if someone with a decent laptop can post timings of a more relevant test? My computer is old and would die before giving me the output of something like : 

```
x = sample(1:100, 3e8, replace = TRUE) # 1 GB

microbenchmark::microbenchmark(
dplyr::case_when(
  x < 10L ~ 0L,
  x < 20L ~ 10L,
  x < 30L ~ 20L,
  x < 40L ~ 30L,
  x < 50L ~ 40L,
  x < 60L ~ 50L,
  x > 60L ~ 60L
)
data.table::fcase(
  x < 10L, 0L,
  x < 20L, 10L,
  x < 30L, 20L,
  x < 40L, 30L,
  x < 50L, 40L,
  x < 60L, 50L,
  x > 60L, 60L
),
times = 5L,
unit ="s")
```